OvernightPrintsSpecTests/FFs/Basket_Feature.feature

```diff
@@ -0,0 +1,300 @@
+		| 1000 Double Sided               | $126.45        | $0.00          | $98.95        |
+	And I see Basket contains "2" elements
+
+	When I remove "500 Double Sided Business Cards" item from basket
+	Then I see Basket Page
+	Then I wait for 10 seconds
+	Then I don't see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+	And I see Basket contains "1" elements
+
+	When I remove "1000 Double Sided" item from basket
+	Then I see Basket Page
+	Then I wait for 10 seconds
+	Then I don't see following product on the Cart
+		| Name                            | Quantity Price | Material Price | Item Subtotal |
+		| 500 Double Sided Business Cards | $39.95         | $0.00          | $29.35        |
+		| 1000 Double Sided               | $126.45        | $0.00          | $98.95        |
+	And I see Basket contains "0" elements
+
+	When I click Log in button on the Header Page
+	And I set following parameters on Login popup
+		| Field         | Value                   |
+		| Email Address | autestomation@gmail.com |
+		| Password      | 4sep98MPcalifUSA        |
+	And I click Log in button on Login popup
+
+	Then I see that user "Test" is logged in
+	And I see Basket Page
+	And I see notification message "Your shopping cart is empty." on the "Basket Page"
+
+Scenario: Check Basket state after decline option "Professional File Review"
+	When I open browser
+	And I navigate to url "https://www.overnightprints.com"
+	And I click CART button on the Header Page
+
+	When I click Continue Shopping button on Basket Page
+	Then I see Main page
+	
+	When I click Create Business Cards button on Main Page
+	And I click Get Started button on "Business Cards" Product page
+	And I click Customize button on "Business Cards" Template page
+	And I click Next button on "Business Cards" Builder page
+	And I click Approve button on "Business Cards" Builder page
+
+	Then I wait for 50 seconds for "Approve Order Page" is load
+	When I click Continue Order button on "Business Cards" Approve Order Page
+
+	Then I see Basket Page
+	And I wait for 10 seconds for "500 Double Sided Business Cards" is load
+	And I see following product on the Cart
+		| Name                            | Quantity Price                  | Item Subtotal  |
+		| 500 Double Sided Business Cards | $39.95                          | $29.35         |
+	And I see Basket contains "1" elements
+	When I click Professional File Review Decline button on Basket Page
+
+	Then I wait for 10 seconds for "Order Summery Block" is load
+	And I see the following properties in the shopping cart
+		| Property             | Value  |
+		| Order SubTotal price | $39.95 |
+		| Total price          | $29.35 |
+
+Scenario Template: Check Basket state after change product parameters on the basket

```

Comment:
Comment 'Need to check total price before and after this action.' from Google Doc has not been fixed.